src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -114,4 +131,30 @@ void notRemoveNonOrphans() {
         // then
         assertThat(authorService.count()).isOne();
     }
+    
+    @Test
+    void notEmptyFirstName() {
+    	// given
+    	Author noFirst = new Author(" ", "Gaarder");
+    	
+    	validator = Validation.buildDefaultValidatorFactory().getValidator();
```

Comment:
Could you please move this up to the constructor so that we only do this once (also remove it out of the other test)?